### PR TITLE
Adopt Laminas CI workflow

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: .laminas-ci.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "extensions": ["mongodb"]
```
Comment (Slamdunk) [inline .laminas-ci.json:2]: Mmm, no this is wrong, `mongo` was correct but not available within `ondrej` apt source for PHP 7.
What should we do here @boesing ?

Comment (boesing) [reply .laminas-ci.json:2]: We've used `mongofill/mongofill` previously. This might work for this PR but we wont be able to provide support for PHP 8.0.

Comment (Slamdunk) [reply .laminas-ci.json:2]: Let's wait for https://github.com/laminas/technical-steering-committee/blob/35bb511a73259d5f30533eb4bffc90793b756915/meetings/agenda.md#abandoning-laminas-cache-satellites-part-2  then. Maybe this work can be dropped altogether.

**Comment by boesing** [comment]
As of todays TSC meeting, we are dropping support for this package.

**Pull request closed without merging**